en/index.md

```diff
@@ -14,62 +14,60 @@ This guide contains everything you need to assemble, configure, and safely fly a
 
 :::warning
 This guide is for the _development_ version of PX4 (`main` branch).

```

Comment:
FYI @sfuhrer @MaEtUgR 

If you happen to be around, would appreciate your review of just this entry point doc. Is it "not wrong". Specifically I say things like "if you add a payload or new hardware you should retune". But should I be more specific about what kind of tuning (e.g. is autotune sufficient) or should I also recommend running the whole tune cycle again? 

I'll be doing a little more tomorrow and merging.

Reply:
I still stand by my experience that with normal payload swaps usually you don't need to retune the vehicle. Of course if you add 2kg far away from the center of gravity then you are changing how it flies but chances are that you then also do that out of specs and can only reach a stay in the air state with an adapted tuning.